### ISSUE TITLE
Explaning the workflow and aim of the project

### ISSUE THREAD

**Issue by Bharadwajshivam28** [developer-diganta/Dino#160]
Hello sir can the workflow of project and aim of this project can be added in README file . Like how the project works and what is the aim of project

**Comment by developer-diganta** [comment]
Yup sure! Also can you tell me if you are either participating in SWOC or IWOC?

**Comment by Bharadwajshivam28** [comment]
> Yup sure! Also can you tell me if you are either participating in SWOC or IWOC?
> 

Yes in iwoc

**Comment by developer-diganta** [comment]
Cool! Do ask me if you need any help!

**Comment by Bharadwajshivam28** [comment]
> Cool! Do ask me if you need any help!

Ok sir

**Comment by Bharadwajshivam28** [comment]
@developer-diganta

**Comment by Bharadwajshivam28** [comment]
@developer-diganta Please look to it sir
